### PR TITLE
Added `ReflectionProperty#getType()` to psalm type stubs for PHP 7.4+

### PR DESCRIPTION
**Pull request by Ocramius**
Ref: https://github.com/Ocramius/ProxyManager/pull/585 (fails build there, noticed missing types)

### CODE REVIEW

File: src/Psalm/Internal/CallMap_74_delta.php [hunk 1]
```diff
@@ -21,5 +21,6 @@
 'old' => [
     'password_hash' => ['string|false', 'password'=>'string', 'algo'=>'int', 'options='=>'array'],
     'password_needs_rehash' => ['bool', 'hash'=>'string', 'algo'=>'int', 'options='=>'array'],
+    'ReflectionProperty::getType' => ['?ReflectionType'],
```
Comment (weirdan) [inline src/Psalm/Internal/CallMap_74_delta.php:24]: I think it should be in the `new` section, as it was added, not removed.

**Comment by Ocramius** [comment]
Ooof, just noticed my mistake there - indeed, should be in "new", but will
fix tomorrow.
Marco Pivetta

http://twitter.com/Ocramius

http://ocramius.github.com/

**Comment by weirdan** [comment]
> but will fix tomorrow.

That won't (and didn't) break anything, 7.4 delta is ignored anyway 😈

**Comment by weirdan** [comment]
> 7.4 delta is ignored anyway 

Fixed in #3094

File: src/Psalm/Internal/CallMap_74_delta.php (hunk 1 above)
Comment (weirdan) [reply src/Psalm/Internal/CallMap_74_delta.php:24]: Also you need to add this to `CallMap.php` itself. Deltas are used to patch the callmap for older versions.

Comment (Ocramius) [reply src/Psalm/Internal/CallMap_74_delta.php]: Adjusted both here and in `CallMap.php` :+1:

**Comment by muglug** [comment]
Thanks!

**Comment by Ocramius** [comment]
@muglug would it be possible to cut a patch release with this in? Not sure if `master` is in releasable state.

**Comment by muglug** [comment]
I'm still working out the ergonomics of `mixed` unions inside arrays, as there are a number of false-positives in an internal [PHPUnit method](https://github.com/sebastianbergmann/phpunit/blob/30d5173bd35d51c48e9ed75b5732d567e0916ae6/src/TextUI/TestRunner.php#L122) that passes in an array that actually should be an object.

**Comment by muglug** [comment]
Basically this: https://psalm.dev/r/d4e71b8666

**Comment by muglug** [comment]
Remaining issues have been ironed out, a release should happen within the next 24 hours

**Pull request merged**